shell.sh

```
@@ -0,0 +1,34 @@
+
+count=0
+while [ $count -lt 5 ]; do
+    echo "Count is: $count"
+    count=$((count + 1))
+done
+
+mkdir /some/directory
+
+current_dir=`pwd`
+
+echo "This is the unset variable: $unset_var"
```

Shellcheck

[warning] unset_var is referenced but not assigned (did you mean 'unused_var'?).

(SC2154)